modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/web/security/identity/propagation/deployment/IdentityPropagationServlet.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2019, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.jboss.additional.testsuite.jdkall.present.web.security.identity.propagation.deployment;
+
+import org.jboss.additional.testsuite.jdkall.present.web.security.identity.propagation.IdentityPropagationAuthenticationTestCase;
+import org.jboss.as.test.integration.web.security.identity.propagation.deployment.Hello;
```

Comment:
This should probably be org.jboss.additional.testsuite.jdkall.present.web.security.identity.propagation.deployment.Hello;?